.csslintrc

```diff
@@ -20,6 +20,7 @@
     "qualified-headings": false,
     "regex-selectors": false,
     "selector-max-approaching": false,
+	"selector-max": false,
```

Comment:
Ok will fix
Yes the compiled CSS has exceeded the 4095 selector. After reading, that should only causing an issue for IE9 and older version of IE.

Also, that means that some clean up in the CSS for all features would need to completed in a near future.